composer.json

```diff
@@ -15,15 +15,15 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.4",
+        "php": ">=7.4|>=8.1",
         "ext-curl": "*",
         "ext-tidy": "*",
         "fossar/htmlawed": "^1.2.8",
         "guzzlehttp/psr7": "^2.0",
         "j0k3r/graby-site-config": "^1.0.147",
         "j0k3r/httplug-ssrf-plugin": "^2.0",
-        "j0k3r/php-readability": "^1.2.9",
-        "monolog/monolog": "^1.18.0|^2.3",
+        "j0k3r/php-readability": "dev-master",
```

Comment:
Is there any reason you are bumping readability?

Reply:
Because of `psr/log`, it's detailed in the PR description

Reply:
Because readability 1x only supports psr/log 1.

Reply:
Should I try 2.0.2?

Reply:
Maybe you can put `^2.0` it'll be enough for the `psr/log` v3